### DISCUSSION TITLE
Drift rate noise parameter

### DISCUSSION THREAD

**Discussion by filippoferrari** [mwshinn/PyDDM#80] [Ask for help] [answered]
Hi, 

I have used DMAT in the past and in there you have a parameter which correspond to the inter trial standard deviation of the drift rate. 
Is the class `pyddm.models.noise.NoiseConstant` the same thing as that? Or does this noise affect other parameters as well? 
In case it does, is there a way to fit only the noise of the drift rate?  

Thanks!

**Comment by mwshinn** [comment, marked as answer]
Unfortunately, PyDDM does not have good support for variation in drift rate.  There is still a way to make it work, but it is kind of a hack.  See: https://pyddm.readthedocs.io/en/latest/cookbook/driftnoise.html#drift-uniform for an example with the uniform distribution instead of the normal distribution.

**Reply by Jiashun97** [reply]
Hi Max,

I just implemented the uniform distribution of the drift rate, but it does not work very well with the data. I would like to try the normal distribution and I was wondering if you have any idea on this. 

Does the hyperparameter 'resolution' mean anything in this case? How can I discretize the normal distribution of drift rate? 

Thanks you in advance!

Best regards,
Jiashun

**Reply by mwshinn** [reply]
In the documentation example, DriftUniform spaces the drift rates evenly, i.e., the k'th quantiles of the uniform distribution.  If you want to do a normal distribution, you need to space them around the mean according to the quantile plot of the normal distribution.  (But just to warn you... the model will look almost exactly the same as it does with the uniform distribution, so don't expect any qualitatively different results!)

**Reply by Jiashun97** [reply]
Ah I see. The discretization was the quantiles. I will try to implement this. Thanks a lot.
